### ISSUE TITLE
Escort formation is broken

### ISSUE THREAD

**Issue by onpon4** [naev/naev#1762]
Escorts do not go into formation when created / launched and instead use only the default follow "formation" (i.e. no formation). To see this, equip a ship with some number of fighter bays (I used Lancelot fighter bays but any should do), set them to a key, and launch them. They should start in a circle formation, but they presently don't.

🕵️

**Comment by onpon4** [comment]
After some investigation, I see exactly why formations aren't working: while going to a formation's positions is handled by followers, setting those positions relative to the leader is handled by the leader. Problem is, the only code that actually currently exists to do this is in ai/tpl/generic.lua, via the following function:

```lua
function lead_fleet ()
   if #ai.pilot():followers() ~= 0 then
      if mem.formation == nil then
         formation.clear(ai.pilot())
         return
      end

      local form = formation[mem.formation]
      if form == nil then
         warn(string.format(_("Formation '%s' not found"), mem.formation))
      else
         form(ai.pilot())
      end
   end
end
```

That this is the source is proven if you have a fleet out and do `player.pilot():control()` in the Lua console. This immediately causes the ships to go into formation because when you do that, the player's ship becomes AI-controlled.

So the solution to this bug is to do that exact same thing as the function above for the player every frame (albeit with some changes so it doesn't use the AI API), and the question is where this can be done and where it was done before.

🕵️

**Comment by bobbens** [comment]
Fixed in 1178d6f6df01eec4d92bc8f5a1cf11714cf9dd68 . Thanks!
